diff --git a/package/homebrew/magnum-integration.rb b/package/homebrew/magnum-integration.rb
index f49f5a0..0f7ca18 100644
--- a/package/homebrew/magnum-integration.rb
+++ b/package/homebrew/magnum-integration.rb
@@ -16,7 +16,7 @@ class MagnumIntegration < Formula
   def install
     system "mkdir build"
     cd "build" do
-      system "cmake", "-DCMAKE_BUILD_TYPE=Release", "-DCMAKE_INSTALL_PREFIX=#{prefix}", "-DWITH_BULLET=ON", "-DWITH_DART=ON", "-DWITH_EIGEN=ON", "-DWITH_GLM=ON", "-DWITH_IMGUI=OFF", ".."
+      system "cmake", "-DCMAKE_BUILD_TYPE=Release", "-DCMAKE_INSTALL_PREFIX=#{prefix}", "-DWITH_BULLET=ON", "-DWITH_DART=ON", "-DWITH_EIGEN=ON", "-DWITH_GLM=ON", "-DWITH_IMGUI=ON", "-DIMGUI_DIR=IMDIR", ".."
       system "cmake", "--build", "."
       system "cmake", "--build", ".", "--target", "install"
     end
