--- package/debian/rules	2019-09-13 15:31:09.374620101 -0400
+++ package/debian/rules	2019-09-13 15:29:39.626622653 -0400
@@ -15,7 +15,9 @@
 override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCMAKE_BUILD_TYPE=Release \
-		-DWITH_BULLET=ON
-		-DWITH_DART=OFF
-		-DWITH_GLM=ON
-		-DWITH_IMGUI=OFF
+		-DWITH_BULLET=OFF \
+		-DWITH_DART=OFF \
+		-DWITH_GLM=ON \
+		-DWITH_IMGUI=ON \
+		-DIMGUI_DIR=IMDIR
+
